screens/Contacts/ContactListScreen.tsx

```diff
@@ -0,0 +1,125 @@
+  {uid: '13', firstName: 'Def', lastName: 'Steph', profilePictureUrl: '', description: '-', interests: ['movies'], qualifications: ['history']},
+  {uid: '14', firstName: 'Hij', lastName: 'Non', profilePictureUrl: '', description: 'Mataphisical question', interests: [], qualifications: []},
+]
+
+export const ContactListScreen = () => {
+  const [filteredContacts, setFilteredContacts] = useState(dummyData);
+  const [searchText, setSearchText] = useState('');
+  const [selectedTab, setSelectedTab] = useState('Plain View');
+ 
+  const handleSearch = (text: string) => {
+    setSearchText(text);
+    if(text){
+        const filtered = dummyData.filter((contact) =>
+          `${contact.firstName} ${contact.lastName}`.toLowerCase().includes(text.toLowerCase())
+        )
+        setFilteredContacts(filtered);
+    }else{
+        setFilteredContacts(dummyData);
+    }
+  }
+
+  const RenderOneContact = ({ item } : { item : Contact }) => (
+    <View style={styles.cardContainer}>
+      {item.profilePictureUrl? (
+        <Image
+          style={styles.profilePicture}
+          source={{uri: item.profilePictureUrl}}
+        />
+      ) : (
+        <View style={styles.profilePicture}>
+          <Ionicons name="person" size={50} color="black" />
+        </View>
+      )}
+      <View style={styles.container}>
+        <Text 
+          style={styles.contactDescription}
+          numberOfLines={3}>
+            {item.description}
+        </Text>
+        <View style={styles.container}>
+          <Text style={styles.contactName}>{item.firstName}</Text>
+          <Text style={styles.contactFriendType}>friend</Text>
+        </View>
+      </View>
+    </View>
+  )
+
+  return (
+    <View style={styles.container}>
+      
+      <View style={styles.horizonatlBarContainer}>
+        <Text>Here will be the top navigation bar</Text>
+      </View>
+      
+      <View style={styles.viewChoiceContainer}>
+        <TouchableOpacity style={styles.contactList} onPress={() => setSelectedTab('Plain View')}>
+          <Text style={[styles.greyLightText, selectedTab === 'Plain View' && styles.darkBoldText]}>
+            Plain View
+          </Text>
+          <View style={[styles.viewCHoiceUnderLine, selectedTab === 'Graph View' && styles.invisibleBackground]}/>
+        </TouchableOpacity>
+
+        <TouchableOpacity style={styles.contactList} onPress={() => setSelectedTab('Graph View')}>
+          <Text style={[styles.greyLightText, selectedTab === 'Graph View' && styles.darkBoldText]}>
+            Graph View
+          </Text>
+          <View style={[styles.viewCHoiceUnderLine, selectedTab === 'Plain View' && styles.invisibleBackground]}/>
+        </TouchableOpacity>
+      </View>
+
+      <View>
+        <TextInput 
+          style = {styles.searchBar}
+          placeholder="Search..." 
+          value={searchText}
+          onChangeText={handleSearch}
+        />
+      </View>
```

Comment:
for the simple <View> without styling you can use <></> it more concise (but it's small details)